pom.xml

```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>it.eng.idsa</groupId>
 	<artifactId>multipart-message-processor</artifactId>
-	<version>1.0.14-SNAPSHOT</version>
+	<version>1.0.16-SNAPSHOT</version>
```

Comment:
Any special reason why we skip one version?

Reply:
probably was .15 on my machine and forgot to check with master branch, I will fix it